docs/source/wave.rst

```diff
@@ -0,0 +1,131 @@
+==============
```

Comment:
Awesome documentation.

Reply:
@aerorahul I don't know if this will be integrated with the rest of the RTD documentation as this is a new file. Do I need to do anything for that?   I did check to make sure it looked okay on the github interface so hopefully the formatting will look okay at least.

Reply:
Here's a preview: https://global-workflow--1608.org.readthedocs.build/en/1608/

I think we need to link to the wave documentation from existing docs, otherwise there is no way to get there.

Reply:
(The wave file specifically: https://global-workflow--1608.org.readthedocs.build/en/1608/wave.html)